### DISCUSSION TITLE
Music-Assistant card

### DISCUSSION THREAD

**Discussion by PrzemekSkw** [music-assistant/support#415] [Feature requests and ideas]
It will be great option to have card for favorites radio stations from Music-Assistant or Artists from local folder. 
Regards.

**Comment by erkr** [comment]
I move this request to the discussions, thanks for filing.

**Reply by OzGav** [reply]
You could use a [button card](https://www.home-assistant.io/dashboards/button/) which calls a script. You could also use any number of custom components that also call a script. So not sure of the need for a dedicated card just for favourites?

**Comment by OzGav** [comment]
Closing as this is a duplicate of https://github.com/orgs/music-assistant/discussions/954 and whilst this came first it hasn’t seen many comments. This is also related https://github.com/orgs/music-assistant/discussions/2601
